overlay.nix

```diff
@@ -1,10 +1,24 @@
 self: super: {
+
   # https://github.com/nix-community/poetry2nix/issues/218
   poetryOverrides = self: super: {
     typing-extensions = super.typing-extensions.overridePythonAttrs
       (old: { buildInputs = (old.buildInputs or [ ]) ++ [ self.flit-core ]; });
 
-    lief = super.lief.overridePythonAttrs (old: { dontUseCmakeConfigure = true; nativeBuildInputs = [ self.pkgs.cmake ]; });
+    # This is an unreleased version of Lief that fixes a bug when generates GNU notes
+    # https://github.com/lief-project/LIEF/commit/72ebe0d89e94c18d2b64da2cbbc7a0a0d53a5693
+    lief = super.lief.overridePythonAttrs (old: {
+      version = "0.12.72ebe0d";
+      src = super.pkgs.fetchFromGitHub {
+        owner = "lief-project";
+        repo = "LIEF";
+        rev = "72ebe0d89e94c18d2b64da2cbbc7a0a0d53a5693";
+        sha256 = "039fwn6b92aq2vb8s44ld5bclz4gz2f9ki2kj7gy31x9lzjldnwk";
+      };
+      enableParallelBuilding = true;
+      dontUseCmakeConfigure = true;
+      nativeBuildInputs = [ self.pkgs.cmake ];
+    });
```

Comment:
For whatever reason in `nix build` this still built with a single core...
Building Lief surprisingly took quite a while.

Reply:
Trying to parse this, what does "dontUseCmakeConfigure" do in this context?

Reply:
Oh that's interesting.  Maybe lief has both cmake and poetry available, poetry is nice but it usually doesn't build a ton of c/c++, so it might be kinda slow without poking.